### PR TITLE
PUBDEV-7141: aucpr updates etc for 3.28.0.1

### PR DESCRIPTION
**Pull request by hannah-tillman**

### CODE REVIEW

File: h2o-docs/src/product/automl.rst [hunk 1]
```diff
@@ -68,8 +68,13 @@ Optional Miscellaneous Parameters
 
 - `max_runtime_secs_per_model <data-science/algo-params/max_runtime_secs_per_model.html>`__: Specify the max amount of time dedicated to the training of each individual model in the AutoML run. Defaults to 0 (disabled). Note that setting this parameter can affect AutoML reproducibility.
 
+- **modeling_plan**: The list of modeling steps to be used by the AutoML engine (they may not all get executed, depending on other constraints).
+
+- `monotone_constraints <data-science/algo-params/monotone_constraints.html>`__: A mapping that represents monotonic constraints. Use +1 to enforce an increasing constraint and -1 to specify a decreasing constraint. Note that constraints can only be defined for numerical columns. This option can only be used when the distribution is either gaussian or bernoulli.
+
 -  `stopping_metric <data-science/algo-params/stopping_metric.html>`__: Specify the metric to use for early stopping. Defaults to ``AUTO``. The available options are:
     
+    - ``aucpr``
```
Comment (seb-h2o) [inline h2o-docs/src/product/automl.rst]: even if not the first lexicographically, shouldn't we leave `AUTO` on top of the list?

Comment (seb-h2o) [reply h2o-docs/src/product/automl.rst]: same below

File: h2o-docs/src/product/automl.rst [hunk 2]
```diff
@@ -68,9 +68,14 @@ Optional Miscellaneous Parameters
 
 - `max_runtime_secs_per_model <data-science/algo-params/max_runtime_secs_per_model.html>`__: Specify the max amount of time dedicated to the training of each individual model in the AutoML run. Defaults to 0 (disabled). Note that setting this parameter can affect AutoML reproducibility.
 
+- **modeling_plan**: The list of modeling steps to be used by the AutoML engine (they may not all get executed, depending on other constraints).
```
Comment (angela0xdata) [inline h2o-docs/src/product/automl.rst]: - **modeling_plan**: The list of modeling steps to be used by the AutoML engine. (They may not all get executed, depending on other constraints.)

File: h2o-docs/src/product/automl.rst [hunk 3]
```diff
@@ -68,9 +68,14 @@ Optional Miscellaneous Parameters
...
+
+- `monotone_constraints <data-science/algo-params/monotone_constraints.html>`__: A mapping that represents monotonic constraints. Use +1 to enforce an increasing constraint and -1 to specify a decreasing constraint. Note that constraints can only be defined for numerical columns. This option can only be used when the distribution is either gaussian or bernoulli.
```
Comment (angela0xdata) [inline h2o-docs/src/product/automl.rst]: Remove "Note that constraints can only be defined for numerical columns. This option can only be used when the distribution is either gaussian or bernoulli."

File: h2o-docs/src/product/data-science/algo-params/monotone_constraints.rst [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ``monotone_constraints``
 ------------------------
 
-- Available in: GBM, XGBoost
+- Available in: AutoML, GBM, XGBoost
 - Hyperparameter: no
 
 Description
```
Comment (angela0xdata) [inline h2o-docs/src/product/data-science/algo-params/monotone_constraints.rst]: This topic currently has a note: 
**Note**: This option can only be used when the distribution is either ``gaussian`` or ``bernoulli``.

Update that note to:
**Note**: In GBM and XGBoost, this option can only be used when the distribution is either ``gaussian`` or ``bernoulli``.

File: h2o-docs/src/product/automl.rst [hunk 5]
```diff
@@ -30,7 +30,7 @@ Required Stopping Parameters
 
 One of the following stopping strategies (time or number-of-model based) must be specified.  When both options are set, then the AutoML run will stop as soon as it hits one of either of these limits. 
 
-- `max_runtime_secs <data-science/algo-params/max_runtime_secs.html>`__: This argument specifies the maximum time that the AutoML process will run for, prior to training the final Stacked Ensemble models. If neither ``max_runtime_secs`` nor ``max_models`` are specified by the user, then ``max_runtime_secs`` defaults to 3600 seconds (1 hour).
+- `max_runtime_secs <data-science/algo-params/max_runtime_secs.html>`__: This argument specifies the maximum time that the AutoML process will run for, prior to training the final Stacked Ensemble models. The default is 0 (no limit), but dynamically sets to 1 hour if none of ``max_runtime_secs`` and ``max_models`` are specified by the user.
```
Comment (ledell) [inline h2o-docs/src/product/automl.rst]: Thanks, this is better worded...

**Pull request merged**